nixos/modules/services/x11/display-managers/sddm.nix

```diff
@@ -219,8 +219,6 @@ in
         # Load themes from system environment
         QT_PLUGIN_PATH = "/run/current-system/sw/" + pkgs.qt5.qtbase.qtPluginPrefix;
         QML2_IMPORT_PATH = "/run/current-system/sw/" + pkgs.qt5.qtbase.qtQmlPrefix;
```

Comment:
These two seem like they should also be handled. `QT_PLUGIN_PATH` is set, but to something different. `QML2_IMPORT_PATH` isn't set. Not sure if this is suspicious or not.

Reply:
I'm not familiar with these things for QT, so I choose to not change this so the PR doesn't get blocked. Looking at git blame it was intentional a while ago, so I'd have to check with @ttuegel.

Reply:
`QT_PLUGIN_PATH` and `QML2_IMPORT_PATH` should not be set system-wide because the paths have version-specific prefixes. For historical reasons, they actually are set system-wide at the moment, but it is important that SDDM keep these locally-hardcoded paths.

Reply:
Thanks, good that I didn't try to introduce a change to remove this.